internal/applicationsnapshot/report.go

```diff
@@ -289,6 +309,67 @@ func generateMarkdownSummary(r *Report) ([]byte, error) {
 	return markdownBuffer.Bytes(), nil
 }
 
+func generateTextReport(r *Report) ([]byte, error) {
+
+	mainTemplate := hd.Doc(`
```

Comment:
Perhaps use [embed](https://pkg.go.dev/embed) and have the template in a separate file?

Reply:
Yep, that's done in the latest revision.